### PR TITLE
RFTools Spawner, Solars, Food, Crafts and other fixes and improvments

### PR DESCRIPTION
**Pull request by Krutoy242**
Bunch of fixes and suggestions. Found while making E2E addon.

### Configs:

- [Market.json](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-045106270700c5fc3393efad36efa71dR62-R81) Rustic seeds and saplings in market
- [enderiomachines.cfg](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-25f3d12f0f499e86414f53f9d8e2c535R152-R165) Higher EnderIO Solar Panels production, because they can be made only after achieving Alloy Smelter (tier4 machine)
- [enderiopowertools.cfg](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-4985ec75b125e96e280d232025dfcaf4R21-R36) x4 Higher EnderIO Capacitor's storages, so they better compete with Induction Matrixes (still capped with int max, duh)
- [nuclearcraft.cfg](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-1bf67099d5b52d282f0e718c20ff4739R776-R779) About x10 times higher NC solar panels to compete with much cheaper Mekanism's. Also disabled tile shielding (because radiation is disabled)
- [dimlets.json](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-598178ba20dc0fb7589cb238deb0d5dc) Removed Flat Colored Blocks from dimlets. They flooded dimlet workbench too much being basically cobblestone. Also added bunch of blocks from TE, AA, EU2, NC, EIO and RedstoneArsenal mods
- [rftools.cfg](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-244b3c11a3d7d5b3a38ffc0af4cdbe0cR163-R175) Higher coal generator production. Bit higher than Petrified Fuel Generator from IF because of next tier machine case
- [rftools.cfg](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-244b3c11a3d7d5b3a38ffc0af4cdbe0c) **39** of new Living Matter from mods, mostly Animania & IceAndFire. **267** new entities for Spawner, with estimated costs (higher than usual spawn cost). [Gist for better readability](https://gist.github.com/Krutoy242/ae6889b308b062e331d6d4bc2cb50c93)
- [solcarrot.cfg](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-9c2b602751fea38bcb03698cbccc8eb4R74-R110) Expand table of levels, so additional heart gain not capped with 600 uniq food

### Scripts

- [ActuallyAdditions.zs](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-b96ddcfd11ee5c4af6c926e75521ef25R222-R228) Uncommon Ores for Laser
- [AkashicTome.zs](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-230b1b2267d798cde45eaaa89013511eR24-R29) New manuals in Super Akashic Tome from **TiC**, **Solcarrot** and **Rustic**
- [Cyclic.zs](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-d3589a1057ddb86e0af0891abcababd9R126-R133) Harder recipe for explosion proof glass
- [EnderIO.zs](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-06c578d1164b678df2bde6d43c619287R211-R230) Simplified crafters from EIO. They still slower than RFTool's ones, and can do only one recipe. Normal version still requires capacitor. Also, simplified Power Monitor.
- [IndustrialForegoing.zs](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-05972e88f00be1b7a1631e3905ff07f1R154-R169) Simlified Ore Dictionary machines, because they serve technical purposes and required in early game because of wrong loot from chests and dragon nests.
- [NuclearCraft.zs](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-1e432b93f967addf8d73b4cbab3899d4R49-R56) Fixed recipe of basic battery (was Elite Plating, changed to Basic)
- [NuclearCraft.zs](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-1e432b93f967addf8d73b4cbab3899d4R49-R56) Disabled smelting of unused EnderIO alloys
- [OreDict.zs](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-6f49fe271efc9bd8794282c9d92baed8R110-R118) Added BoP dirt to ore:dirt to prevent weird situation when you need to run searching for actual dirt if spawned in salty biome.
- [TinkersConstruct.zs](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-1a9aedc131316a77bf589e09608cababR150-R155) Added missed Blood sling recipe
- [Tooltips.zs](https://github.com/Krutoy242/Enigmatica2Expert/commit/7ea5fb07c5275089b66565f4397bc6abbd2bcdf1#diff-e3c73b6c878981eeb662a131e2a4289cR523-R570) Ingame visual representation of replicable items in UU-Replicator (being one of FAQ in discord)

### Some screenshots:

New dimlets:
New dimlets

New Spawner Entities:
New Spawner Entities

New Books:
New Books

Missed Blood Sling recipe:
Missed Blood Sling recipe

List of items that can be replicated in JEI:
List of items that can be replicated in JEI

### CODE REVIEW

**Comment by NillerMedDild** [comment]
Wow, this must've taken you a ton of time, thanks a lot for your contribution! I'll look everything over and merge before the next update :)

File: config/enderio/enderiomachines.cfg [hunk 1]
```diff
@@ -149,20 +149,20 @@ generator {
         B:helmetChargeOthers=true
 
         # Energy generated per TICK by Simple Photovoltaic Panels. (synced from server) [range: 1 ~ 2147483647, default: 10]
-        I:solarPanel0Gen=10
+        I:solarPanel0Gen=200
```
Comment (NillerMedDild) [inline config/enderio/enderiomachines.cfg]: The Solar panels have been buffed too much imo, I would prefer the values to be:
* 50
* 200
* 800
* 3200

I know they're expensive, but they are fully passive and very easily stackable.

File: config/nuclearcraft.cfg [hunk 2]
```diff
@@ -773,10 +773,10 @@ generators {
 
     # RF/t generated. Order: Basic, Advanced, DU, Elite.
     I:solar_power <
-        20
-        60
-        180
-        540
+        100
```
Comment (NillerMedDild) [inline config/nuclearcraft.cfg]: I would like this changed to
* 40
* 160
* 640
* 2560

**Comment by Krutoy242** [comment]
I made requested changes, committed and resolve comments. Its my first pull request ever, so i don't know what to do next.

**Comment by NillerMedDild** [comment]
Merged, thanks a ton for your contribution, really great stuff :D

**Comment by NillerMedDild** [comment]
I forgot to switch to master before publishing to CurseForge, so your PR won't be in the pack until 1.79 sadly :(

**Comment by Krutoy242** [comment]
> I forgot to switch to master before publishing to CurseForge, so your PR won't be in the pack until 1.79 sadly :(

Oh, unlucky

**Pull request merged**